lumigator/python/mzai/backend/backend/config_templates.py

```diff
@@ -66,6 +66,24 @@
 
 # Inference templates
 
+default_infer_template = """{{
+    "name": "{job_name}/{job_id}",
+    "dataset": {{ "path": "{dataset_path}" }},
+    "pipeline": {{
```

Comment:
Is "pipeline" the right name for this config? It's a very generic name; I assume HF uses it because it is composed of several serial stages.
However, I admit I cannot come up with a suitable replacement :-/

Reply:
@javiermtorres that is something I'm not sure either. If someone else has a better name I'm all ears.

Reply:
This are all parameters that are specific to HF pipelines, right? How does `hf_pipeline` sound?
We might also want to be aware of the fact that if we introduce a new pipeline (e.g. text generation) we might have some pipeline-specific params we want to provide. Would we put them in here within some `task_specific_params` section as they do in HF?

Reply:
@aittalam sure, we can integrate it with [this](https://github.com/mozilla-ai/lumigator/blob/99ce0bf59d7ee365acc365df2f0287b44172291e/lumigator/python/mzai/backend/backend/models.yaml). However, I would like to address than in a different PR. Is that okay?

I'll also rename `pipeline` to `hf_pipeline`.